vpr/src/place/move_generators/move_generator.h

```diff
@@ -157,9 +159,13 @@ class MoveGenerator {
                                               double delta_c,
                                               float timing_bb_factor);
 
+  public:
+    static float first_rlim;
```

Comment:
I'd suggest adding some comments above it to explain the purpose of this variable and why it is defined as a static variable.